packages/siop-oid4vp/lib/__tests__/IT.spec.ts

```diff
@@ -141,7 +141,7 @@ function getVCs(): IVerifiableCredential[] {
   return vcs
 }
 
-describe('RP and OP interaction should', () => {
+describe.skip('RP and OP interaction should', () => {
```

Comment:
Please do not just skip tests without making a reference to a ticket

Reply:
fixed